### DISCUSSION TITLE
Hover text on blocks

### DISCUSSION THREAD

**Discussion by shawnboy777** [voidlabs/mosaico#632] [Q&A]
This is simply a question.  How can I change the default text that shows when hovering over the blocks (where it says ""Click or drag here to add this block to the template.")?  I would like, instead, to have a basic description for each of what the block is so users can determine if it is the appropriate block for their template.

Thanks in advance.

**Comment by bago** [comment]
Blocks are defined in the master template, so every text must be defined there.
IIRC there is an "help" property that you can define in the @supports -ko-blockdefs and it is only used for properties. You can define it for "blocks", but then you'll have to work in the mosaico code to see how to bring this help in the tooltip you say (it is defined in "src/tmpl/toolbox.tmpl.html" but I don't know if the blockDef you find in the "foreach: blockDefs" gives you access to the "help" property (I think NO), so you probably have to change some code so that the help properties ends up there.

**Comment by shawnboy777** [comment]
Thanks for the reply!  I will look into it.
